README.md

```diff
@@ -80,5 +80,17 @@ Once CMake has generated the build files, within `<WorkSpace Dir>/build` run...
 - [Build] `cmake --build .` or `ninja -v` (if the build generator was Ninja)
 - [Test] `ctest --parallel`
 
+Documentation
+--------------
+**Latest release**

```

Comment:
Is this a placeholder? If so, remove it. Let's have a section for Releases and a section for "other" links.